### DISCUSSION TITLE
ICONS cannot be switched properly

### DISCUSSION THREAD

**Discussion by ghost** [breezy-weather/breezy-weather#548] [Q&A]
Huawei HarmonyOS
The app icon does not automatically change with the weather
Other similar third party applications can also change ICONS normally

**Comment by papjul** [comment]
Works fine on my side:
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/6a9f6b7f-5568-4b6c-af2f-c0568f98e818)
